### PR TITLE
Add more meaningful fixtures.

### LINKED ISSUE #420: Update fixtures with more meaningful information
Including working project URL's

### PR DESCRIPTION
**Pull request by henricattoire**
Closes #420. 

Projects Seaspiration, Bike To Work and Destination Insight show off Markdown features.

- Jim Carrey has a comment on his suggestion to Seaspiration.
- Florence Nightingale has a comment on her suggestion to Destination Insight.
- Jan Pieter has a comment on his suggestion by Friedrich.
- Florence Nightingale has a comment on her suggestion by Friedrich.

Let me know is this is good enough to demo. If not, I can always add more even more meaningful fixtures.

### CODE REVIEW

File: be/osoc/common/fixtures/fixtures.json [hunk 1]
```diff
@@ -378,25 +378,25 @@
...
       "first_name": "Lisa",
       "last_name": "Simpsons",
       "call_name": "",
       "email": "lisa@simpsons.com",
       "phone_number": "",
       "language": "dutch",
-      "motivation": "My parents forced me",
-      "cv": "http://example.com",
-      "portfolio": "http://example.com",
+      "motivation": "My parents thinks its time for me to start working.",
```
Comment (leveys) [inline be/osoc/common/fixtures/fixtures.json:397]: typo: thinks -> think and its -> it's

**Review by leveys — APPROVED** [review]
ziet er goed uit!

**Review by Wouter01 — APPROVED** [review]
Nice additions!

**Comment by Wouter01** [comment]
Can you change the colors of marketer and storyteller to [valid](https://quasar.dev/style/color-palette) colors?

**Review by Wouter01 — APPROVED** [review]

**Pull request merged**